lib/org/chromium/webidl/IDLFileContents.js

```diff
@@ -32,5 +31,11 @@ foam.CLASS({
       required: true,
       final: true,
```

Comment:
Let's drop `final`. I *think* I made the `DatastoreDAO` play nice with `final` values, but their setters fail silent (even in dev mode). Seems risky.